pkg/config/config.go

```diff
@@ -292,55 +303,99 @@ func httpClientForIssuer(iss OIDCIssuer) (*http.Client, error) {
 }
 
 func (fc *FulcioConfig) prepare() error {
-	if _, ok := fc.GetIssuer("https://kubernetes.default.svc"); ok {
-		// Add the Kubernetes cluster's CA to the system CA pool, and to
-		// the default transport.
+	fc.verifiers = make(map[string][]*verifierWithConfig, len(fc.OIDCIssuers))
+	for _, iss := range fc.OIDCIssuers {
+		if err := fc.insertVerifier(iss); err != nil {
+			log.Logger.Errorf("error creating provider for issuer URL %q: %v", iss.IssuerURL, err)
+			continue
+		}
+	}
+
+	_, verifierPresent := fc.verifiers[k8sIssuerURL]
+	k8sIssuer, issuerConfigured := fc.GetIssuer(k8sIssuerURL)
+	if issuerConfigured && !verifierPresent {
+		// configure static validator for k8s that cover metaIssuers
```

Comment:
Can you explain more why the k8s issuer would be present in the config but not already have its provider/verifier created in the loop above? This check doesn't look necessary to me.